### PR TITLE
[MOBILE-903] Release 8.1

### PR DESCRIPTION
**Pull request by BrianBatchelder**
### What do these changes do?
Release 8.1 of the Cordova Plugin

### Why are these changes necessary?
Need to support EUCS.

### How did you verify these changes?
Manual testing of both iOS and Android sample apps

### Anything else a reviewer should know?
`jsdoc` was updated to 3.6.3 to address a security vulnerability flagged by Github and npm.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,12 +43,15 @@
...
+    "jsdoc": "3.6.3",
     "minami": "1.2.3",
     "semver": "^5.6.0",
     "superagent": "^4.1.0"
   },
   "scripts": {
     "generate-docs": "jsdoc ./www/UrbanAirship.js -t ./node_modules/minami/ -r jsdoc_readme.md"
+  },
+  "dependencies": {
+    "cordova": "^9.0.0"
```
Comment (rlepinski) [inline package.json]: whats this about?

Comment (BrianBatchelder) [reply package.json]: It was added automatically. I accepted it since we state the same in our README.

File: www/UrbanAirship.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 /* Copyright Urban Airship and Contributors */
 
+/**
+ * @module UrbanAirship
```
Comment (marc-scig) [inline www/UrbanAirship.js]: does this moving of the `@module` directive change the output of the documentation in any way, or is it just a style thing?

File: package.json (hunk 1 above)
Comment (rlepinski) [reply package.json]: Its probably the right thing but lets hold off for a bigger cordova release

File: www/UrbanAirship.js (hunk 2 above)
Comment (BrianBatchelder) [reply www/UrbanAirship.js]: It fixed one thing that seemed to break with the JSDoc update. The other, less important thing will need a ticket.

File: package.json (hunk 1 above)
Comment (BrianBatchelder) [reply package.json]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+==================================
+Version 8.1.0 - September 23, 2019
+==================================
+
+- Updated Android Airship SDK to 10.1.3
+- Updated iOS Airship SDK to 11.1.2
+- Added support for EU data center
```
Comment (rlepinski) [inline CHANGELOG.md]: cloud site

**Pull request merged**